bazel/envoy_mobile_repositories.bzl

```diff
@@ -105,9 +106,10 @@ def swift_repos():
 def kotlin_repos():
     http_archive(
         name = "rules_java",
-        sha256 = "ddc9e11f4836265fea905d2845ac1d04ebad12a255f791ef7fd648d1d2215a5b",
-        strip_prefix = "rules_java-5.0.0",
-        url = "https://github.com/bazelbuild/rules_java/archive/refs/tags/5.0.0.tar.gz",
+        sha256 = "19462d64b1586c0d4ea0e87f9325be2514f0eb84e56dbf3245450451b3701581",
+        strip_prefix = "rules_java-43243982abc76390ef64be62379a1353f9011771",
+        # TODO(jpsim): Switch back to bazelbuild repo when https://github.com/bazelbuild/rules_java/pull/65 is merged
```

Comment:
@Bencodes do you have a better idea here? Do you think it's ok to use this reverted branch for now to unblock?

Some details as to the error we're hitting in https://github.com/bazelbuild/rules_java/pull/64